apps/users/src/app/app.controller.ts

```diff
@@ -190,17 +191,25 @@ export class UserController {
   }
 
   @CastcleAuth(CacheKeyName.SyncSocial)
-  @Get('syncSocial')
+  @Get('sync_social')
```

Comment:
For URL naming, we should use dashes (-) to separate keywords and use plural nouns for defining resources.

```suggestion
  @Get('sync-socials')
```